test/WebSites/TagHelpersWebSite/TagHelpers/AddProcessedAttributeTagHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace TagHelpersWebSite.TagHelpers
+{
+    [TargetElement("input")]
+    public class AddProcessedAttributeTagHelper : TagHelper
```

Comment:
This `TagHelper` ensures that the input elements in `UnboundDynamicAttributes.cshtml` are run over by a `TagHelper` but not bound by one. Also adding the `processed` attribute lets us visually see that the test is working as expected.